DESCRIPTION

```diff
@@ -31,16 +31,30 @@ Depends:
     teal.transform (>= 0.5.0.9015)
```

Comment:
Teal requirement needs to be bumped with https://github.com/insightsengineering/teal/pull/1432

Possibly to `teal (>= 0.15.2.9091),`

Keeping conversation opened until the PR is closed

Reply:
I approved that PR, so working on this PR can continue.